Assets/HoloToolkit/Sharing/Prefabs/ManualIpConfigUtility.prefab

```diff
@@ -0,0 +1,3126 @@
+%YAML 1.1
```

Comment:
I see the benefit, but I think that's something devs should do in their own project.